### PR TITLE
Add troubleshooting

### PR DESCRIPTION
**Pull request by toofff**
Adding troubleshooting file for recurring questions.

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Troubleshooting
+
+This is a list of common pitfalls on using API Platform, and how to avoid them.
+
+## Using Docker
+
+If the web container cannot starting and display this error `Error starting userland proxy: Bind for 0.0.0.0:80`, this is the port 80 is already used.
```
Comment (Simperfit) [inline troubleshooting.md]: `If the web container cannot start and display this `Error starting userland proxy: Bind for 0.0.0.0:80`, it means that the port 80 is already used.`

**Comment by toofff** [comment]
@Simperfit Done

**Comment by toofff** [comment]
@dunglas Done

File: troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Troubleshooting
+
+This is a list of common pitfalls on using API Platform, and how to avoid them.
+
+## Using Docker
+
+If the web container cannot start and display this `Error starting userland proxy: Bind for 0.0.0.0:80`, it means that the port 80 is already used.
+
+The configuration docker is planned to be launch on the port 80 in the docker-compose.yml file.
```
Comment (dunglas) [inline troubleshooting.md]: `docker-compose.yml`

**Comment by toofff** [comment]
@dunglas Done

**Comment by dunglas** [comment]
Thanks @toofff

File: troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Troubleshooting
+
+This is a list of common pitfalls on using API Platform, and how to avoid them.
+
+## Using Docker
+
+If the web container cannot start and display this `Error starting userland proxy: Bind for 0.0.0.0:80`, it means that the port 80 is already used.
+
+The configuration docker is planned to be launch on the port 80 in the `docker-compose.yml` file.
```
Comment (teohhanhui) [inline troubleshooting.md]: This sentence is confusing.

**Comment by teohhanhui** [comment]
I'm really not sure we need this in our documentation. This is just Docker / Docker Compose basics.

At least it should be moved to the `distribution` docs.

**Pull request merged**